web/src/components/merchant/listings/index.tsx

```diff
@@ -16,76 +16,20 @@
 
 import React from 'react';
 
-import CentralisedContainer from 'components/common/CentralisedContainer';
-import MerchantSideBar from 'components/common/MerchantSideBar';
+import MerchantPage from 'components/common/MerchantPage';
 import EmptyListingsPlaceholder from 'components/merchant/listings/EmptyListingsPlaceholder';
-import Row from 'muicss/lib/react/row';
 import {useLocation} from 'react-router-dom';
-import styled from 'styled-components';
 
-const PageContainer = styled.div`
-  min-height: 100vh;
-  height: 100%;
-  width: 100%;
-
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-  align-items: center;
-`;
-
-const PageContent = styled(CentralisedContainer)`
-  height: 100vh;
-  width: 100%;
-  max-width: none;
-  margin: 0;
-  padding: 0 5%;
-`;
-
-const HeaderRow = styled(Row)`
-  width: 100%;
-  margin-left: 0;
-  margin-right: 0;
-  margin-top: 5%;
-  margin-bottom: 1%;
-
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-  align-items: flex-end;
-`;
-
-const Header = styled.h1`
-  font-weight: bolder;
-`;
-
-const ListingsBody = styled(CentralisedContainer)`
-  height: 100%;
-  width: 100%;
-`;
-
-const OngoingListingsPage: React.FC = () => {
```

Comment:
Why do you delete the OngoingListingPage?

Reply:
@xlai20 I refactored out the common parts. Then, I forgot to change the name to `ListingsPage` from `AddListingPage`.